xml/System.Net.Configuration/WindowsAuthenticationElement.xml

```diff
@@ -71,6 +71,8 @@ By default, Windows allocates a separate SSPI handle for each <xref:System.Net.H
 </system.net>
 ```
 
+If this behavior impacts performance, you can use the `defaultCredentialsHandleCacheSize` to use a single cached SSPI credential handle for default network credentials. You can do this by setting this property to a value greater than 0. The exact value depends on application requirements. Generally, a value between 50 and 100 works for most high performant applications.
```

Comment:
```suggestion
If this behavior impacts performance, set <xref:System.Net.Configuration.WindowsAuthenticationElementDefaultCredentialsHandleCacheSize> to a value greater than 0 to use a single, cached SSPI credential handle for default network credentials. The exact value depends on application requirements. Generally, a value between 50 and 100 works for most high-performant applications.
```

Reply:
I guess I'm not clear if the user should set the value via the property or in the config file.

Reply:
>I guess I'm not clear if the user should set the value via the property or in the config file.

Oops. I should not have used the word "property".  There is no API (class, property, etc.) to configure this. The only way to enable/disable/configure this feature is via the configuration settings.

I'll incorporate your suggested changes directly since you removed the word 'property'.